### ISSUE TITLE
[Bug?] Particle animation not working

### ISSUE THREAD

**Issue by rger0** [sp614x/optifine#4029]
## Description of Issue
Hi,
I have been trying to animate the angry_villager particle with a resource pack, to no success ;
I followed the guidelines described in custom_animations.txt, but it doesn't seem to work for me ;

## List of concerned files
- located in _assets/minecraft/textures/particle/_ : **angry.png**
- located in _assets/minecraft/optifine/anim/_ : **angry.png** , **angry.properties**

## File resolution
- The **angry.png** file in _assets/minecraft/textures/particle/_ is a 128x128 png file ;
- The **angry.png** file in _assets/minecraft/optifine/anim/_ is a 128x256 png file (2 frames) ;

Here's what the **angry.properties** file looks like :
![code](https://user-images.githubusercontent.com/44302126/83774439-99d1ae80-a67d-11ea-90cb-e160f1383adc.png)

## What happens in game
Basically, when I punch a villager, instead of animating the particle, MC+optifine simply uses the **angry.png file** already present in _assets/minecraft/textures/particle/_

## OptiFine Version
preview_OptiFine_1.15.2_HD_U_G1_pre16

## Installation Method
Standalone installer

## Conclusion
Is my code broken? Am I attempting something that Optifine doesn't do?
Sorry if I did something wrong, I tried a variety of things, and nothing seems to work ;
Any help is appreciated

**Comment by sp614x** [comment]
In 1.15 the particles textures are combined in a texture atlas like the terrain textures.
They can't be animated with custom animations, but ".mcmeta" animations should still work.
